lib/endpoint/find-parameters.js

```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
   })
   return state
 }
+
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
+      if (output.params[i].default === 'Time.now') {
+        output.params[i].default = output.params[i].default.replace('Time.now', '<current date/time>')
+      }
+    }
+    return output
+  })
+  return state
```

Comment:
you can remove both `return` statements, they don’t do anything. You use return for `.map()`, `.forEach()` doesn’t do anything with the returned value